src/bun.js/bindings/webcore/HTTPHeaderMap.cpp

```diff
@@ -236,15 +236,7 @@ String HTTPHeaderMap::get(HTTPHeaderName name) const
 void HTTPHeaderMap::set(HTTPHeaderName name, const String& value)
 {
     if (name == HTTPHeaderName::SetCookie) {
-        auto cookieName = extractCookieName(value);
-        size_t length = m_setCookieHeaders.size();
-        const auto& cookies = m_setCookieHeaders.data();
-        for (size_t i = 0; i < length; ++i) {
-            if (extractCookieName(cookies[i]) == cookieName) {
-                m_setCookieHeaders[i] = value;
-                return;
-            }
-        }
+        m_setCookieHeaders.clear();
```

Comment:
Will this work if you call headers.append? It doesn’t

Reply:
Yeah this will work. headers.append will go through HTTPHeaderMap::add. Set will clear the existing set-cookie values.